custom/icds_reports/migrations/0186_refresh_views.py

```diff
@@ -12,4 +12,4 @@ class Migration(migrations.Migration):
         ('icds_reports', '0179_location_deprecation_columns'),
```

Comment:
What are these changes?

Reply:
I guess, these are @calellowitz migration files. Only 185 is related to this PR